source/features/open-issue-to-latest-comment.tsx

```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoPath, getOwnerAndRepo} from '../libs/utils';
+
+function init(): void | false {
+	if (getRepoPath() === 'issues') {
+		const {ownerName, repoName} = getOwnerAndRepo();
+		select.all<HTMLAnchorElement>(`.js-issue-row a.muted-link[href^="/${ownerName}/${repoName}/issues/"]`).forEach(link => {
+			link.href = `${link.href}#last`;
```

Comment:
You can set `link.hash` and then link to the footer. Refer to the link in the old `jump-to-bottom` feature